test/native-client.test.js

```diff
@@ -40,12 +40,23 @@ describe('NativeClient', function() {
 
       /*
        * pretends to be a connection-model providing every function call
-       * required in NativeClient#connect, but returns topology of our choice
+       * required in NativeClient#connect, but returns topology and connection
+       * params of our choice
        */
-      function mockedConnectionModel(topologyDescription) {
+      function mockedConnectionModel(topologyDescription, connectionOptions) {
```

Comment:
Here you pass connectionOptions.

Reply:
Ah, thanks for pointing that out! Yeah, this was not done for any specific reason, they are the same, and being more consistent in the name would definitely will be better. Will update it all to be `connectionOptions` 👍